integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOCILoadBalancer.java

```diff
@@ -0,0 +1,321 @@
+  private static String domain1Namespace = null;
+  private static String domain1Uid = "lboci-domain-1";
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  // constants for creating domain image using model in image
+
+  private static final String IMAGE_NAME = "ocilb-image";
+  private static final String SAMPLE_APP_NAME = "sample-app";
+  private static String cluster1Name = "cluster-1";
+
+  private static LoggingFacade logger = null;
+  private static String loadBalancerIP = null;
+  private static final String OCI_LB_NAME = "ocilb";
+
+  /**
+   * Install and verify operator.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    logger.info("Get a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    logger.info("Get a unique namespace for WebLogic domain1");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain1Namespace = namespaces.get(1);
+
+    logger.info("install and verify operator");
+    installAndVerifyOperator(opNamespace, domain1Namespace);
+  }
+
+  @AfterAll
+  public void tearDownAll() {
+    Kubernetes.deleteService(OCI_LB_NAME, domain1Namespace);
+  }
+
+  /**
+   * Test covers basic functionality for OCI LoadBalancer .
+   * Create omain and  OCI LoadBalancer.
+   * Check that application is accessabale via OCI LoadBalancer
+   */
+  @Test
+  @DisplayName("Test the sample-app app can be accessed"
+      + " from all managed servers in the domain through OCI Load Balancer.")
+  public void testOCILB() throws Exception {
+
+    // create and verify one cluster mii domain
+    logger.info("Create domain and verify that it's running");
+    String miiImage1 = createAndVerifyMiiImage(MODEL_DIR + "/model-singleclusterdomain-sampleapp-wls.yaml");
+    // create docker registry secret to pull the image from registry
+    // this secret is used only for non-kind cluster
+    logger.info("Create docker registry secret in namespace {0}", domain1Namespace);
+    createOcirRepoSecret(domain1Namespace);
+    createAndVerifyDomain(miiImage1, domain1Namespace, domain1Uid);
+    int clusterHttpPort = 8001;
+
+    assertDoesNotThrow(() -> installAndVerifyOCILoadBalancer(domain1Namespace,
+        clusterHttpPort, cluster1Name, domain1Uid, OCI_LB_NAME),
+        "Installation of OCI Load Balancer failed");
+    loadBalancerIP = getLoadBalancerIP(domain1Namespace,OCI_LB_NAME);
+    assertNotNull(loadBalancerIP, " External IP for Load Balancer is undefined");
```

Comment:
remove the empty character before "External"

Reply:
done